### PR TITLE
xlsum card preparer, with signature and with debug = False

### PR DESCRIPTION
**Pull request by dafnapension**
fixed the signature issue, by recommitting.
Applied yoav's comments re. debug = False.

### CODE REVIEW

**Comment by dafnapension** [comment]
Hi @yoavkatz ,  I applied here all your comments from  #290, for same branch that I committed then without signature.
Here I signed all, and set debug = False, and added the generated json cards.
Please approve and merge.

**Review by yoavkatz — CHANGES_REQUESTED** [review]
Can you change "text"->"document" and "target"->"summary" 

And also add a "text_type" field with value "text".

This will allow it to reuse other summarization templates (once we add them).

**Comment by dafnapension** [comment]
Hi @yoavkatz , I made the changes you suggested.
Please review and approve, if you see fit, and merge.
Thanks!

File: prepare/cards/xlsum.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            outputs=["summary"],
+            metrics=["metrics.rouge"],
+        ),
+        templates=TemplatesList(
+            [
+                InputOutputTemplate(input_format="{document}", output_format="{summary}"),
+            ]
+        ),
+    )
+    if lang == "english":
```
Comment (yoavkatz) [inline prepare/cards/xlsum.py:38]:
```suggestion
    if lang == langs[0]
```

Comment (yoavkatz) [reply prepare/cards/xlsum.py:38]: The reason is , is that we don't want to add to catalogs cards, before at least one is tested.  And we don't know if English is the first.

Comment (dafnapension) [reply prepare/cards/xlsum.py:38]: Thanks, @yoavkatz , this is a very nice thought, that did not occur to me.
Fixing right away!

Comment (dafnapension) [reply prepare/cards/xlsum.py]: Thanks again, @yoavkatz , fixed this, and also for Amazon from yesterday.
Please see if you can approve now.

**Comment by dafnapension** [comment]
Hi @yoavkatz , I fixed per your suggestion, and updated merge into this branch, to avoid the conflicts that grew up..
Please review, and if you see fit - approve and merge.

**Review by yoavkatz — APPROVED** [review]

**Comment by yoavkatz** [comment]
Approved. You can merge,

**Comment by dafnapension** [comment]
Thank you, @yoavkatz , for all the lessons I learned from you

**Pull request merged**